marklogic-data-hub-central/ui/src/components/queries/managing/manage-query-modal/manage-query.tsx

```diff
@@ -140,6 +141,7 @@ const QueryModal = (props) => {
             zeroState: false,
             sortOrder: query.savedQuery.sortOrder,
             database: searchOptions.database,
+            datasource: "final",
```

Comment:
Any particular reason why we have hardcoded the value of datasource here. Shouldn't this be **searchOptions.datasource** ?